recipes/genview/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "genview" %}
+{% set version = "0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+        url: https://github.com/EbmeyerSt/GEnView/archive/refs/tags/v0.1.tar.gz
+        sha256: b297f5c638e186be8cc2c4317aca73f4a67f7c23e5147b6f81afb01f5a64c595
+
+build: 
+  number: 0
+  noarch: python
+  script: python setup.py install
+
+requirements:
+  host:
+    - pip
+    - python=3.6
```

Comment:
why is this pinned to python 36? Is that needed?

Reply:
I get an error in the build test related to the python argparse module, works with python 3.6 but not higher, so it's needed atm

Reply:
What is this error? Should we not fix it? Python 3.6 is end of life soon, so this version will not work for long.

Reply:
The error was likely due to listing argparse as a dependency. It's been built into python since version 2.7. I presume it only existed as a package to aid in the version 2->3 adoption long ago.

Reply:
Great, thank you for making the commit and fixing the issue!